content/blog/online-privacy.en.md

```diff
@@ -0,0 +1,45 @@
+
+Privacy Badger is available for Chrome, Opera, Firefox, and even Firefox on Android phones.
+
+## uBlock Origin
+
+[uBlock Origin](https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm?hl=en) is one of the most popular ad blocker extensions. It prevents pesky popups and widgets from sending your data to advertisers. uBlock Origin lets you create personalized lists which gives you more control over what ads to block and what ads to allow. It has also been found to effectively block more ads than others while using less memory, so it doesn’t slow down your browsing experience.
+
+Unlike ad blockers, anti-tracking extensions block cookies and protect your online anonymity by preventing tracking and malvertising of third-party advertisers.
+
+uBlock Origin is available for Chrome, Opera, Firefox, and Safari.
+
+## Ghostery
+
+[Ghostery](https://www.ghostery.com/) is similar to Privacy Badger, but with a slightly better interface. It detects cookies on websites you visit and blocks the tracking to speedup page-load times—essentially eliminating clutter while protecting your data. However, Ghostery doesn’t block trackers by default. You’ll need to inspect them from the interface and decide what to do with them. There’s also an option to share your data with Ghostery to improve your experience.
+
+## HTTPS Everywhere
+
+Secure websites use the HTTPS (Hyper Text Transfer Protocol Secure) protocol, which encrypts your credit card details, personal information, and other sensitive data. However, not all websites are HTTPS-enabled, so the next step and piece-de-resistance in your privacy setup should be a good site-securing extension.
+
+[HTTPS Everywhere](https://www.eff.org/https-everywhere) add-on is a joint venture project of Tor browser and EFF. When you install the extension, it automatically rewrites the requests, establishes a connection to SSL, then locates and directs you to the most secure versions of the websites you visit. In case your browser connects you to unsecured websites, HTTPS Everywhere will keep your data private and secure from leaking by redirecting it to an encrypted HTTPS.
+
+## Disconnect
+
+[Disconnect](https://disconnect.me/) is an extension that de-personalizes your browsing without cutting off necessary functionality. Available for mobile and desktops, Disconnect works by blocking various bits of third-party tracking code that poses harm to your computer or privacy. When a site makes a network request to connect to anything besides the site you’re visiting, it won’t be able to load that extra content. Once installed, you’ll notice Disconnect building up a tally in its task bar icon. As with the other extensions, you can customize the control for different sites and even experience faster browsing.
```

Comment:
We could also have an 'advanced users' section that mentions more addons:

* Firefox Containers (and Facebook Container)
* Neat URL
* Smart Referer
* NoScript/uMatrix

You don't have to write that section though, if you don't want. We can always add it later.

Reply:
Yes, We can add it later :+1: